src/types/Newsroom.ts

```diff
@@ -97,9 +97,10 @@ export interface Newsroom extends NewsroomRef {
     custom_data_request_link: string | null;
 
     tracking_policy: Newsroom.TrackingPolicy;
-    cookiepro: {
```

Comment:
This will be a breaking change, so a major version bump will be required.

The old property name will still be there for older API clients. Just not declared in the latest interface.